pkg/tsdb/influxdb/flux/query_models.go

```diff
@@ -83,6 +83,12 @@ func getQueryModelTSDB(query *tsdb.Query, timeRange *tsdb.TimeRange, dsInfo *mod
 		To:   endTime,
 	}
 	model.MaxDataPoints = query.MaxDataPoints
+	if model.MaxDataPoints == 0 {
+		model.MaxDataPoints = 10000 // 10k/series should be a reasonable place to abort!
+	}
 	model.Interval = time.Millisecond * time.Duration(query.IntervalMs)
+	if model.Interval.Milliseconds() == 0 {
+		model.Interval = time.Millisecond // 1ms
```

Comment:
his seems like a very low default interval, we should use the interval defined in the data source settings, and if no interval there default to something like 10s

Reply:
Created an issue for the general problem, https://github.com/grafana/grafana/issues/27469